### PR TITLE
Fix invalid download link, misc CI updates

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: src/Uno.PackageDiff/Program.cs [hunk 1]
```diff
@@ -148,7 +148,11 @@ private static async Task<(string path, NuspecReader nuspecReader)> GetPackage(s
 
 					if (latestStable != null)
 					{
-						var url = $"https://api.nuget.org/v3-flatcontainer/{packagePath}/{latestStable.Version}/{packagePath}.{latestStable.Version}.nupkg";
+						var packageId = packagePath.ToLowerInvariant();
+						var version = latestStable.Version.ToString().ToLowerInvariant();
```
Comment (joelverhagen) [inline src/Uno.PackageDiff/Program.cs]:
```suggestion
						var version = latestStable.Version.ToNormalizedString().ToLowerInvariant();
```

Comment (jeromelaban) [reply src/Uno.PackageDiff/Program.cs]: Thanks for the review! I'm not familiar enough with the APIs of this package. I searched for a direct download link in the APIs, but most of those require a local nuget cache to work with.

Comment (joelverhagen) [reply src/Uno.PackageDiff/Program.cs]: Yeah, your current approach is workable. The `ToNormalizedString()` is important though. Non-normalized versions will not be recognized by the download URL and could yield a 404.

e.g. non-normalized version `1.0.0.0-BETA`. Normalization would yield `1.0.0-BETA`. The string used in the URL would be `1.0.0-beta`.

**Pull request merged**